roles/openshift_facts/tasks/main.yml

```diff
@@ -26,6 +26,23 @@
     msg: "openshift-ansible requires Python 2 for {{ ansible_distribution }}"
   when: ansible_distribution != 'Fedora' and ansible_python['version']['major'] != 2
 
+# Fail early if Atomic and old version of Docker
+- block:
+
+  # See https://access.redhat.com/articles/2317361
+  # and https://github.com/ansible/ansible/issues/15892
+  # NOTE: the "'s can not be removed at this level else the docker command will fail
+  - name: Determine Atomic Host Docker Version
+    shell: 'CURLY="{"; docker version --format "$CURLY{json .Server.Version}}"'
+    register: l_atomic_docker_version
```

Comment:
@ashcrow local testing seems to indicate that:
```
- hosts: localhost
  become: no
  gather_facts: no
  connection: local
  vars:
  tasks:
  - command: docker version --format "{{ version_format }}"
    register: result
    vars:
      version_format: !unsafe '{{json .Server.Version}}'
  - debug: var=result
```
would work to provide the docker version. It could be that I am running 2.2.1 locally, though.